worker/echo/com.redhat.Yggdrasil1.Worker1.echo.service.in

```diff
@@ -1,3 +1,4 @@
 [D-BUS Service]
 Name=com.redhat.Yggdrasil1.Worker1.echo
+User=root
```

Comment:
Is it necessary to run this worker as a root? Or is it necessary to run all workers as a root? Could it be changed in the future? It would be more secure to run service as non-root user.